### PR TITLE
Include Link to Windows installer

### LINKED ISSUE #3466: Add information about the Windows installer for the CLI
There's an installer for Windows, which simplifies the instructions on http://exercism.io/clients/cli/windows

The actual text is defined in `x/docs/md/cli/windows.md`.

The installer lives in https://github.com/exercism/windows-installer, instructions for using it are defined in the README of that repo.

### PR DESCRIPTION
**Pull request by RaptorRCX**
Updated, Fixed the whole mess I made with my original pull request. I went ahead and included a separate branch, and these are the changes I made.

Least I learned more about Git while I was fixing everything. Thats a positive I guess.

Fixes #3466

### CODE REVIEW

**Comment by Raptorian** [comment]
#3481

**Comment by RaptorRCX** [comment]
#3481

**Comment by Insti** [comment]
I like that this PR is simple and does one thing. :+1: 

I don't know anything about the status of the windows installer so I can't really comment on the content. 

cc: @blueelvis

File: x/docs/md/cli/windows.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 ## Windows
 
-Below are installation instructions, using the most common method, the Chocolately Package Manager. For further help and instructions, see the following:
+[There is now a installer for Windows available on GitHub](https://github.com/exercism/windows-installer/releases)
```
Comment (kytrinyx) [inline x/docs/md/cli/windows.md]: Let's remove _now_, since most people reading this page won't care that it wasn't available before.

**Comment by kytrinyx** [comment]
I went ahead and made the change.

**Pull request merged**